<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GymAppQtClass</class>
 <widget class="QMainWindow" name="GymAppQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GymAppQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>441</width>
      <height>261</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="BOol " stdset="0">
     <string/>
    </property>
    <widget class="QWidget" name="Admin_tab">
     <property name="styleSheet">
      <string notr="true">background: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/background.png)</string>
     </property>
     <attribute name="title">
      <string>Admin</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #EBE9F5;
border-radius: 11px;

</string>
      </property>
      <widget class="QPushButton" name="Login_button_ad">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_input_ad">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_input_ad">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Admin login</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Member_tab">
     <property name="styleSheet">
      <string notr="true">background: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/background.png)</string>
     </property>
     <attribute name="title">
      <string>Member</string>
     </attribute>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #EBE9F5;
border-radius: 11px;

</string>
      </property>
      <widget class="QPushButton" name="Login_button_mb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="to_register_button">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>260</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
color: #7D43E2;</string>
       </property>
       <property name="text">
        <string>Register to the gym </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_input_mb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_input_mb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Member login </string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Trainer_tab">
     <property name="styleSheet">
      <string notr="true">background: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/background.png)</string>
     </property>
     <attribute name="title">
      <string>Trainer</string>
     </attribute>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #EBE9F5;
border-radius: 11px;

</string>
      </property>
      <widget class="QPushButton" name="Login_button_tr">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_input_tr">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_input_tr">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>231</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #7F7F7F;
padding: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Trainer login </string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GymAppQt.qrc"/>
  <include location="MyRrecourses.qrc"/>
 </resources>
 <connections/>
</ui>
